pre-train/reader/extract_features.py

```diff
@@ -0,0 +1,94 @@
+import librosa
+import numpy as np
+import glob
+import os
+from multiprocessing import Pool, cpu_count
+import sys
+    
+def extract_mel_spec(filename):
+    y, sample_rate = librosa.load(filename)
+    mel_spec = librosa.feature.melspectrogram(y=y,
+                                              sr=sample_rate,
+                                              n_fft=2048,
+                                              hop_length=512,
+                                              win_length=None,
+                                              window='hann',
+                                              center=True,
+                                              pad_mode='reflect',
+                                              power=2.0,
+                                              n_mels=128,
+                                              fmin=0.0,
+                                              fmax=None,
+                                              htk=False,
+                                              norm=1)
+
+    np.save(file=filename.replace(".wav", ".spec"), arr=mel_spec)
```

Comment:
np.save(file=filename.replace(".wav", ".npy"), arr=mel_spec)

Reply:
@jxzhanggg, `np.save` will automatically append a `.npy` extension, and so this way the file name will be:

`original_filename.spec.npy`

I think this is nice because information about the kind of data (i.e. `spec`) is embedded in the filename

thoughts?

Reply:
https://github.com/jxzhanggg/nonparaSeq2seqVC_code/pull/10/files#r360661129

Reply:
Yep, that's a good choice.